sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/ErrorAttachments.java

```diff
@@ -13,6 +12,7 @@
  * Error attachment utilities.
  */
 @SuppressWarnings("WeakerAccess")
+/* TODO (getErrorAttachment): Re-enable error attachment in javadoc when the feature becomes available. Add @ before link. */
 public final class ErrorAttachments {
```

Comment:
Oops, missed that javadoc thing. Changing it to package level. This is enough not to have `ErrorAttachments` javadoc.